scripts/src/admin/services.ts

```diff
@@ -262,7 +261,28 @@ async function appToHtml(app: Application): Promise<string> {
 </tr>`;
 }
 
-async function offerToHtml(offer: Offer): Promise<string> {
+async function offerToHtml(offer: Offer, appOffer?: AppOffer): Promise<string> {
```

Comment:
@doodyparizada did you add/adjust unit tests?

Reply:
This is the admin page. Admin isn't currently part of the unit tests. It could be when we migrate it to an API service instead of server-side rendering.